nsist/tests/test_configuration_validator.py

```diff
@@ -1,17 +1,75 @@
-from nose.tools import *
+import configparser
 import os
+
+from nose.tools import *
+
 from .. import configreader
-import configparser
+from .. import get_installer_builder_args
+
 
 DATA_FILES = os.path.join(os.path.dirname(__file__), 'data_files')
 
 def test_valid_config():
     configfile = os.path.join(DATA_FILES, 'valid_config.cfg')
-    configreader.read_and_validate(configfile)
+    config = configreader.read_and_validate(configfile)
+    print(config['Application'])
+    print('Application' in config)
+    print(config.has_section('Application'))
+    # assert False
 
 def test_valid_config_with_shortcut():
     configfile = os.path.join(DATA_FILES, 'valid_config_with_shortcut.cfg')
-    configreader.read_and_validate(configfile)
+    config = configreader.read_and_validate(configfile)
+
+def test_valid_config_with_values_starting_on_new_line():
+    configfile = os.path.join(DATA_FILES, 'valid_config_value_newline.cfg')
+    config = configreader.read_and_validate(configfile)
+    sections = ('Application', 'Python', 'Include', 'Build')
+    assert len(config.sections()) == len(sections)
+    for section in sections:
+        assert section in config
+        assert config.has_section(section)
+
+    assert config.get('Application', 'name') == '\nMy App'
```

Comment:
This is very thorough, but checking for all these incorrect values before transforming them seems superfluous - do you want to check for just one or two and then call `get_installer_builder_args`?